prime-router/src/test/resources/fhirengine/smoketest/Expected_HL7_to_HL7_FULLELR.hl7

```diff
@@ -1 +1 @@
OBX|1|SN|28-1^Ampicillin [Susceptibility] by Minimum inhibitory concentration (MIC)^LN^AM^AMPICILLIN^L^2.40^5.67|1|=^16|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L^2.40^5.67||R^Resistant. Indicates for microbiology susceptibilities only.^HL70078^R^R^L^2.5.1^5.67|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^R|1033122247^Chahal^Shalini^^^Dr.^^^RML&&ISO^L^^^FI^^^^^^^^MD
OBX|2|ST|7018-5^Gentamicin.high potency [Susceptibility] by Minimum inhibitory concentration (MIC)^LN^GMS^GENTAMICIN SYNERGY SCREEN^L^2.40^5.67|1|SYN-S|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L^2.40^5.67||S^Susceptible. Indicates for microbiology susceptibilities only.^HL70078^S^S^L^2.5.1^5.67|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^S|1033122247^Chahal^Shalini^^^Dr.^^^RML&&ISO^L^^^FI^^^^^^^^MD
OBX|3|SN|29254-0^Linezolid [Susceptibility] by Minimum inhibitory concentration (MIC)^LN^LZD^LINEZOLID^L^2.40^5.67|1|=^2|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L^2.40^5.67||S^Susceptible. Indicates for microbiology susceptibilities only.^HL70078^S^S^L^2.5.1^5.67|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^S|1033122247^Chahal^Shalini^^^Dr.^^^RML&&ISO^L^^^FI^^^^^^^^MD
OBX|4|ST|6933-6^Streptomycin.high potency [Susceptibility] by Minimum inhibitory concentration (MIC)^LN^STS^STREPTOMYCIN SYNERGY SCREEN^L^2.40^5.67|1|SYN-S|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L^2.40^5.67||S^Susceptible. Indicates for microbiology susceptibilities only.^HL70078^S^S^L^2.5.1^5.67|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^S|1033122247^Chahal^Shalini^^^Dr.^^^RML&&ISO^L^^^FI^^^^^^^^MD
OBX|5|SN|19000-9^Vancomycin [Susceptibility]^LN^VA^VANCOMYCIN^L^2.40^5.67|1|>=^32|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L^2.40^5.67||R^Resistant. Indicates for microbiology susceptibilities only.^HL70078^R^R^L^2.5.1^5.67|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^R|1033122247^Chahal^Shalini^^^Dr.^^^RML&&ISO^L^^^FI^^^^^^^^MD
SPM|1|09339017^M12776123&&&ISO||119303007^Microbial isolate^SCT^ISLT^ISOLATE^L^07/31/2012^5.67||||RA^RIGHT ARM^L^5.67||||||RA|||20280808092805-0600^20280808092805-0600|20210804001800-0600||||||||3.0|^BACTEC BLOOD CULTURE BOTTLES

```

Comment:
The only change here was on the line where we had a CE obx that was dropping data:
`OBX|1|CE||1||`
`OBX|1|CE|77202-0^Laboratory comment [Text] in Report Narrative^LN^VERGPCOM^COMMENT^L^2.40^5.67|1|COM^^L|`

Side note: Not really sure why git/GitHub is viewing this file as a single line. IntelliJ parses the segments as individual lines to problem. I guess something with the eol encoding, if it's fixable I wouldn't mind making that change.